protocol/devops/README.md

```diff
@@ -0,0 +1,75 @@
+# DevOps Protocol
+
+A set of guidelines, checklists and tutorials for deploying web apps in a sane
+and reproduceable way.
+
+## Table of Contents
+
+* [Continuous Integration](#continuous-integration)
+* [Servers](#server-setup)
+* [Static websites](#static-websites)
+* [DNS](#server-setup)
+* [SSL](#server-setup)
+    * [Sabayon Samples](#sabayon-samples)
+
+## Continuous Integration
+
+* Use [SemaphoreCI] ([TravisCI] is also fine for open source, but tends to be
+    slower)
+* Set up automatic deploys to staging (from the `master` branch), and manual
+    deploys to production
+* Ensure the CI server is capable of deploying to all environments, so that no
```

Comment:
to be honest, I hadn't even though of that option when I wrote this (CI deploys is what I'm using on my current project, didn't even give it much thought)

But you mean using something like `heroku pipelines:promote`? (and the GUI equivalent)

I can add a reference to that, but wouldn't mention it as the default one because, AFAIK, we never actually used that as our main deploy tool